### PR TITLE
docs: incorporate Alex feedback

### LINKED ISSUE #124: Documentation for ambiguous aliases
We should add documentation encouraging the user to input a concept id (e.g. uniprot accession) when available instead of a gene symbol to ensure an accurate normalized gene symbol is returned.

### LINKED ISSUE #220: Docs reorg
* Expand the "See the public REST service" sentence on the index page, move it up a bit (Quickstart instructions/description using live service). In general, emphasize usage of the REST endpoint.
* Collapse quick/full to combined local install page
* Clean up the README

### PR DESCRIPTION
**Pull request by jsstevenson**
Wasn't really sure what to title this commit, just following up on some things from a 1 on 1 a while back. 

* close #220 , combine installation pages, emphasize public REST API usage, move detailed instructions out of README
* Also close #124 , add usage instructions about what kinds of search terms work best
* Add reference to @anastasiasmith1221 slides. As that work continues to develop, we can update as needed (steal figures etc)
* Add source metadata description
* Remove citations (unnecessary, though it was a helpful exercise -- if we ever need to do this again, refer back to commit history here)
* add a bit more data to the live normalize figure
* misc cleanup

TODO: figure out why API ref isn't working

### CODE REVIEW

File: README.md [hunk 1]
```diff
@@ -1,188 +1,75 @@
...
-* Detect AWS Credentials
-* Detect Private Key
-
-Before first commit run:
-
-```commandline
-pre-commit install
+```python
+>>> from gene.database import create_db
+>>> from gene.query import QueryHandler()
```
Comment (korikuzma) [inline README.md:62]:
```suggestion
>>> from gene.query import QueryHandler
```

**Review by korikuzma — CHANGES_REQUESTED** [review]
small change

File: docs/source/normalizing_data/sources.rst [hunk 2]
```diff
@@ -9,6 +9,8 @@ Normalized records are constructed by aggregating data from established resource
 
    Per the `GA4GH Variation Representation Specification (VRS) <https://vrs.ga4gh.org/en/stable/>`_, Genomic and cytogenic locations are stored and represented as `VRS locations <https://vrs.ga4gh.org/en/stable/terms_and_model.html#location>`_.
 
+TODO do not merge before adding this part: describe source metadata
```
Comment (jsstevenson) [inline docs/source/normalizing_data/sources.rst:12]: todo: address this

File: docs/source/usage.rst [hunk 3]
```diff
@@ -6,9 +6,9 @@ Overview
```
Comment (jsstevenson) [inline docs/source/usage.rst:6]: todo -- the module/method link to create_db in this page isn't working. address this

**Comment by jsstevenson** [comment]
staging version up at https://gene-normalizer--242.org.readthedocs.build/en/242/

**Review by korikuzma — APPROVED** [review]

**Pull request merged**